modules/metrics/README.md

```diff
@@ -25,7 +25,7 @@ A Meter metric, per status-code, tagged with:
 |-------------|----------------------------|------------------------------------------------------|
 | what        | "endpoint-request-rate"    | Enable/disable with ENDPOINT_REQUEST_RATE            |
 | status-code | *                          | "200", "404", "418", etc.                            |
-| unit        | "request"                  |                                                      |
+| unit        | "request/s"                | Requests per second.                                 |
```

Comment:
I noticed that internally for us this is "message/s". Am I correct in assuming the open-source module without Spotify-flavoring will have "request/s" as its unit?